e2e/cypress/integration/enterprise/system_console/compliance/download_compliance_export_file_spec.js

```diff
@@ -0,0 +1,208 @@
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @enterprise @system_console
+
+import * as TIMEOUTS from '../../../../fixtures/timeouts';
+
+describe('Compliance Export', () => {
+    before(() => {
+        cy.apiRequireLicenseForFeature('Compliance');
+        cy.apiUpdateConfig({
+            MessageExportSettings: {
+                DownloadExportResults: true,
+            },
+        });
+    });
+
+    it('MM-T3435 - Download Compliance Export Files - CSV Format', () => {
+        // # Go to compliance page and enable export
+        gotoCompliancePage();
+        enableComplianceExport();
+
+        // # Navigate to a team and post an attachment
+        gotoTeamAndPostImage();
+
+        // # Goto compliance page and start export
+        gotoCompliancePage();
+        exportCompliance();
+
+        // # Get the download link
+        cy.get('@firstRow').findByText('Download').parents('a').should('exist').then((fileAttachment) => {
+            const fileURL = fileAttachment.attr('href');
+
+            // # Download the file
+            downloadAttachmentAndVerifyItsProperties(fileURL);
+        });
+    });
+
+    it('MM-T3438 - Download Compliance Export Files when 0 messages exported', () => {
+        // # Go to compliance page and enable export
+        gotoCompliancePage();
+        enableComplianceExport();
+
+        // # Navigate to a team and post an attachment
+        gotoTeamAndPostImage();
+
+        // # Goto compliance page and start export
+        gotoCompliancePage();
+        exportCompliance();
+
+        // # Get the download link
+        cy.get('@firstRow').findByText('Download').parents('a').should('exist').then((fileAttachment) => {
+            const fileURL = fileAttachment.attr('href');
+
+            // # Download the File
+            downloadAttachmentAndVerifyItsProperties(fileURL);
+
+            // # Export compliance again
+            exportCompliance();
+
+            // # Download link should not exist this time
+            cy.get('.job-table__table').
+                find('tbody > tr:eq(0)').
+                findByText('Download').should('not.exist');
+        });
+    });
+
+    it('MM-T3439 - Download Compliance Export Files - S3 Bucket Storage', () => {
+        // # Goto file storage settings Page
+        cy.visit('/admin_console/environment/file_storage');
+        cy.get('.admin-console__header', {timeout: TIMEOUTS.ONE_MIN}).should('be.visible').and('have.text', 'File Storage');
+
+        // # Get AWS credentials
+        const AWS_S3_BUCKET = Cypress.env('AWS_S3_BUCKET');
+        const AWS_ACCESS_KEY_ID = Cypress.env('AWS_ACCESS_KEY_ID');
+        const AWS_SECRET_ACCESS_KEY = Cypress.env('AWS_SECRET_ACCESS_KEY');
+
+        // # Config AWS settings
+        cy.findByTestId('FileSettings.DriverNamedropdown').select('amazons3');
```

Comment:
@anurag6713 
Following are the suggestions as per Automation standard, but these are not blocker and all test cases are passed so approving it.

`cy.findByRole('combobox', { name: /file storage system:/i }).select('amazons3');`

Reply:
Thanks Malik! I will replace them.